### PR TITLE
Change Batch & Stream Sources to Updated Dataset

### PR DESCRIPTION
**Pull request by maheshmurag**

### CODE REVIEW

File: fraud/data_sources/transactions.py [hunk 1]
```diff
@@ -39,19 +39,12 @@ def raw_data_deserialization(df):
...
-    database='demo_fraud_v2',
-    table='transactions',
-    timestamp_field='timestamp',
-    datetime_partition_columns=partition_columns,
-    # Data in demo_fraud_v2.transactions lands slightly after the real wall time, e.g. the partition for Nov 4 may not
-    # land until Nov 5 00:30:00. Delay materialization jobs by one hour to accommodate this upstream data delay. This
-    # delay will be reflected in training data generation for batch feature views to reflect its impact on online
-    # freshness.
-    data_delay=timedelta(hours=1),
+batch_config = FileConfig(
```
Comment (nickaustinlee) [inline fraud/data_sources/transactions.py:42]: This FileConfig does not specifically identify the file to load. Would this cause Tecton to try and load every parquet file in this directory? Maybe this is why GHF takes a long time?

Comment (maheshmurag) [reply fraud/data_sources/transactions.py:42]: good callout - i just tested with a notebook and didn't observe any speed up with/without specifying the exact filepath

**Comment by nickaustinlee** [comment]
**Summary of Fixes and QA Results**
- Existing cross-account IAM role “arn:aws:iam::706752053316:role/tecton-demo-fraud-data-cross-account-kinesis-ro” was amended to include access to the new lightweight kinesis stream. Access to the legacy kinesis stream was unchanged, so any prior MFT environments and future MFT environments can still refer to the legacy stream if needed. 
- Fixed small bug in fraud_users data source definition. The timestamp field was changed from nonexistent "timestamp" to the correct "signup_timestamp"
- Confirmed that "tecton apply" works successfully on this entire repo. Built it totally new in a test live workspace, with Tecton 0.7.3
- Confirmed that new stream source works in Tecton by calling start_stream_preview() on the new data source. 
- I tested all Streaming Fraud FVs with online=True and offline=True and monitored the backfill and forwardfill jobs on Tecton UI. Jobs were successful. The entire bootstrap backfill was done in approximately 10 minutes! 

Based on the above, I believe this code is ready to ship out to future MFTs, pending final review.

**Comment by nickaustinlee** [comment]
I tested a deployment of the MFT codebase to the Coinbase MFT and confirmed it works, after adjusting the bucket policies to allow anybody on earth to perform ListBucket and GetObject on the /tutorials folder (which should be public anyway). The bucket and folder is this one: s3://tecton.ai.public/tutorials/ (owned by the tecton-production aws account)

I also went into the Coinbase Databricks account and successfully ran a .start_stream_preview() on it (the instance profile on the cluster must be the default tecton-spark-node profile) on the new Kinesis data source. I have confirmed the batch backfill jobs were successful. 

Everything suggests this is ready for go-live! Just merge your PR and it is good to go.

**Pull request merged**